docs/overrides/main.html

```
@@ -1,13 +1,18 @@
-{% extends "base.html" %} {% block announce %}
+{#- HSSM ecosystem shared announcement banner.
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)